lib/storage/providers/AsyncStorage.js

```diff
@@ -73,6 +73,13 @@ const provider = {
      */
     removeItem: AsyncStorage.removeItem,
 
+    /**
+     * Remove given keys and their values from storage
+     * @param {Array} keys
+     * @returns {Promise}
+     */
+    removeItems: keys => Promise.all(_.map(keys, key => AsyncStorage.removeItem(key))),
```

Comment:
This is required because we're using `AsyncStorage` as the mock provider for `NativeStorage` in unit tests, so `removeItems` needs to be defined here as well